metadata/packagesruntime.json

```diff
@@ -2020,7 +2020,497 @@
         "command": "./run-kotor1.sh",
         "engine_name": "reone"
     },
-    "39510": {
+    "34270": {
+    "game_name": "SEGA Mega Drive & Genesis Classics",
+    "download": [
+        {
+            "name": "blastem",
+            "url": "https://github.com/luxtorpeda-dev/packages/releases/download/blastem/",
+            "file": "blastem-34270.tar.xz"
+        }
+    ],
```

Comment:
The tabbing isn't correct, as in the game_name is on the same spacing/tabbing as the app id